oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -24,10 +24,23 @@ option java_package = "oak.session.noninteractive.v1";
 message PublicKeyInfo {
```

Comment:
I assume this is only the message between the untrusted launcher and the guest, is that correct? The trusted part will not have most of these fields.